### DISCUSSION TITLE
2023/04/27/form-designer-naiveui/

### DISCUSSION THREAD

**Discussion by giscus** [SX-Code/blog-comments#14] [Announcements]
# 2023/04/27/form-designer-naiveui/

form-create-designer-naiveui是基于@form-create/naive-ui vue3版本实现的表单设计器组件。可以通过拖拽的方式快速创建表单，提高开发者对表单的开发效率，节省开发者的时间。

https://sx-code.github.io/2023/04/27/form-designer-naiveui/

**Comment by heshiqi1** [comment]
厉害

**Comment by heshiqi1** [comment]
1
